### PR TITLE
compile custom customer sass from site configuration

### PR DESCRIPTION
**Pull request by grozdanowski**
This addition to code checks if the current scss file that's included and needs to be compiled in is called `customer-sass-input`. If so, replace the contents of the file with the value of `customer_sass_input` from the customers site configuration.

### CODE REVIEW

File: openedx/core/djangoapps/site_configuration/models.py [hunk 1]
```diff
@@ -200,6 +200,8 @@ def _formatted_sass_variables(self):
     def _sass_var_override(self, path):
         if 'branding-basics' in path:
             return [(path, self._formatted_sass_variables())]
+        if 'customer-sass-input' in path:
+            return [(path, self.values.get('customer_sass_input', ''))]
```
Comment (OmarIthawi) [inline openedx/core/djangoapps/site_configuration/models.py]: @grozdanowski So, if `branding-basics` exists, this function will `return` early and `customer-sass-input` won't be used. I think we want to use both, and here's the function that does it:

```python
def _sass_var_override(self, path):
    overrides = []

    if 'branding-basics' in path:
        overrides.append(
            (path, self._formatted_sass_variables(),)
        )

    if 'customer-sass-input' in path:
        overrides.append(
            (path, self.values.get('customer_sass_input', '')),
        )

    return overrides
```

I have not tested it, but you're welcome to.

### TL;DR;

This function is called by `compile_microsite_sass` and eventually used as the [`importers`](https://github.com/sass/libsass-python/blob/0.10.0/sass.py#L311-L313) parameter by [`compile_sass`](https://github.com/appsembler/edx-platform/blob/appsembler/amc/master/openedx/core/djangoapps/appsembler/sites/utils.py#L84-L91)).

cc: @melvinsoft

Comment (grozdanowski) [reply openedx/core/djangoapps/site_configuration/models.py]: @OmarIthawi I have both branding-basics and customer-sass-input and it compiles both. I believe the function iterates over every import.

Comment (OmarIthawi) [reply openedx/core/djangoapps/site_configuration/models.py]: Ok! Apparently I'm missing something as I still don't really understand how it works. But we can move with the code as-is.

**Comment by grozdanowski** [comment]
Merging.
